### PR TITLE
[B+C] Call event when animal tempted by player. Adds BUKKIT-4162

### PR DESCRIPTION
**Pull request by yoo2001818**
**The Issue**:

When player tempts animal, the event doesn't occur.

**Justification for this**:

Will call EntityTargetLivingEvent when player tempts animal.

**PR Breakdown**:

In function a(), seems like setting the target, added Event caller.

**JIRA Tickets**:

BUKKIT-4162 - https://bukkit.atlassian.net/browse/BUKKIT-4162

### CODE REVIEW

File: src/main/java/org/bukkit/event/entity/EntityTargetEvent.java [hunk 1]
```diff
@@ -116,6 +116,10 @@ public static HandlerList getHandlerList() {
          */
         DEFEND_VILLAGE,
         /**
+         * When the target tempts the entity.
+         */
+        TEMPTED,
```
Comment (riking) [inline src/main/java/org/bukkit/event/entity/EntityTargetEvent.java:121]: "TEMPTED" may not be a good name for this. Maybe "lure"?

Comment (yoo2001818) [inline src/main/java/org/bukkit/event/entity/EntityTargetEvent.java:121]: I named to 'tempted' because class name is 'PathfinderGoalTempt'.

Comment (riking) [inline src/main/java/org/bukkit/event/entity/EntityTargetEvent.java:121]: We do _not_ follow ~~vanilla naming conventions~~ **the naming conventions you see in the unmodified vanilla source**. We are to pick names that make the most sense.
For a good example: `DEFEND_VILLAGE` above. If villagers ever get weapons in addition to Iron Golems, that could be applied to them. 
For a bad example: `PIG_ZOMBIE_TARGET` above. The naming of it is specific to zombie pigmen - but normal zombies now have the same "group attack" behavior.

To be clear, future expansion is not the only goal - others include readability and obviousness. I do not think TEMPTED is a good name. Can you think of another?

**Comment by md-5** [comment]
> vanilla is obfuscated and the naming convention is decided by the bukkit team

**Comment by turt2live** [comment]
@yoo2001818 If you are still interested in this please update it to the latest code base and update the PR description. The justification, description, and overview are a bit lacking. Please also provide testing materials as per the CONTRIBUTING.md guidelines.

The mentioned changes will also need to be performed.

**Comment by turt2live** [comment]
Currently this PR is missing the following:
- Testing materials / Proof of concept (just to show that this indeed works)
- The ability to be pulled (this may be needed after the 1.7 update)
- The above mentioned concerns need to be addressed.

If anyone can tackle these tasks, the chance of this PR getting pulled significantly increases.

**Comment by turt2live** [comment]
Closing this due to inactivity and lack of testing. If anyone feels as though they can tackle the issue, please open a new PR addressing the concerns listed above.

Thanks!

**Pull request closed without merging**